html-Detector/SubSystemExample/databaseQueries/index.html

```diff
@@ -0,0 +1,387 @@
+              with
+              the ID <code>tableContainer</code>.
+            </p>
+            <div id="tableContainer" class="demo-container mdl-shadow--2dp">
+              <button onclick="runDbTableDemo()"
+                class="mdl-button mdl-js-button mdl-button--raised mdl-button--colored">
+                Run dbTable Demo
+              </button>
+              <div id="dbTableOutput" class="mdl-typography--body-1"></div>
+            </div>
+
+            <script>
+              function runDbTableDemo() {
+                import('/includes/tooldaq.js').then(({ dbTable }) => {
+                  const query = "SELECT * FROM monitoring LIMIT 10";
+                  const headers = ["Time", "Device", "Data"];
+
+                  dbTable(query, headers).then(data => {
+                    console.log("dbTable result:", data);
+                    displayTable(data, headers);
+                  }).catch(error => {
+                    document.getElementById('dbTableOutput').innerHTML = `<p style="color:red;">Error: ${error.message}</p>`;
+                  });
+                });
+
+                function displayTable(data, headers) {
+                  const container = document.getElementById("tableContainer");
+                  container.innerHTML = ""; // Clear previous content
+
+                  if (!data || data.length === 0) {
+                    container.innerHTML = "<p>No data available.</p>";
+                    return;
+                  }
+
+                  let table = document.createElement("table");
+                  table.classList.add("mdl-data-table", "mdl-js-data-table", "mdl-shadow--2dp");
+
+                  // Create table header
+                  let thead = document.createElement("thead");
+                  let headerRow = document.createElement("tr");
+                  headers.forEach(header => {
+                    let th = document.createElement("th");
+                    th.textContent = header;
+                    th.classList.add("mdl-data-table__cell--non-numeric");
+                    headerRow.appendChild(th);
+                  });
+                  thead.appendChild(headerRow);
+                  table.appendChild(thead);
+
+                  // Create table body
+                  let tbody = document.createElement("tbody");
+                  data.forEach(row => {
+                    let tr = document.createElement("tr");
+                    row.forEach(cell => {
+                      let td = document.createElement("td");
+                      td.textContent = cell;
+                      td.classList.add("mdl-data-table__cell--non-numeric");
+                      tr.appendChild(td);
+                    });
+                    tbody.appendChild(tr);
+                  });
+                  table.appendChild(tbody);
+
+                  container.appendChild(table);
+                }
+              }
+            </script>
+            <hr>
+          </div>
+
```

Comment:
Again, no point in `dbHtmlTable` since the user can write it trivially by combining `dbTable` (or `dbJson`) and `makeTable`.

If you still want to add this function, add its description to `tooldaq.mkd` and to the top of `tooldaq.js`.